next.config.js

```diff
@@ -0,0 +1,13 @@
+require('dotenv').config();
```

Comment:
Hey look, dotenv. Ask me for the `.env` file in Slack so you can run it locally.